frontend/src/features/admin-form/share/ShareFormModal.tsx

```diff
@@ -201,7 +208,17 @@ export const ShareFormModal = ({
       return
     } catch (err) {
       setClaimGoLoading(false)
-      setGoLinkHelperText(goLinkClaimFailureHelperText)
+
+      let errMessage =
```

Comment:
Hmm is this error message not a constant somewhere in our code base yet? 👀 
Oh, its not, interesting HAHA

Reply:
😅 perhaps let me make all 3 errors a constant in the ShareFormModal itself

Reply:
Oh this is a very minor nit btw, don't worry abt it! Was just wondering lel

Reply:
[b9ebcd9](https://github.com/opengovsg/FormSG/pull/6714/commits/b9ebcd948a2395f29eb5cac71b4f32f4037c7535)

I think still good to now have magic floating texts haha so abstracted them